samples/cpp/benchmark_app/main.cpp

```diff
@@ -131,8 +131,6 @@ ov::hint::PerformanceMode get_performance_hint(const std::string& device, const
                 ov_perf_hint = ov::hint::PerformanceMode::LATENCY;
             } else if (FLAGS_hint == "cumulative_throughput" || FLAGS_hint == "ctput") {
                 ov_perf_hint = ov::hint::PerformanceMode::CUMULATIVE_THROUGHPUT;
-            } else if (FLAGS_hint == "none") {
```

Comment:
Now benchmark app does not allow user to set hints and nstreams/nthreas/pin at the same time. So "UNDEFINED" are useful in current logic when user want to set nstreams/nthreas/pin. 

If we need to keep this logic after remove "UNDEFINED", need additional logic and help message for it.

Reply:
revert changes in BA, will be done separatelly